voice/bxml/callbacks/transferAnswer.md

```diff
@@ -1,19 +1,39 @@
 {% method %}
 ##  Transfer Answer Event – <Transfer> verb
 When processing a [`<Transfer>`](../verbs/transfer.md) verb, this event is sent when a called party (B-leg) answers.  The event is sent to
-  the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered.  [`<PlayAudio>`](../verbs/playAudio.md) and/or [`<SpeakSentence>`](../verbs/speakSentence.md) verbs returned by this callback will be
-  executed for the called party only. Afterward, the called party will be bridged to the original
-  call.
+  the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered. Valid BXML returned by this callback will be executed for the called party only. Afterward, the called party will be bridged to the original
+  call. 
   
-  It is important to note that no other BXML verbs may be specified after a Transfer Answer Event is called.
+  Valid Bxml must be made up of executable verbs. The list of verbs that are executable when returned from this callback is as follows: 
+
+
+| Verb                                              | Executable    |
+|---------------------------------------------------|---------------|
+| [Conference](../verbs/conference.md)              | No            |
+| [Forward](../verbs/forward.md)                    | No            |
+| [Gather](../verbs/gather.md)                      | Yes           |
+| [Hangup](../verbs/hangup.md)                      | Yes           |
+| [Pause](../verbs/pause.md)                        | Yes           |
+| [PauseRecording](../verbs/pauseRecording.md)      | Yes           |
+| [PlayAudio](../verbs/playAudio.md)                | Yes           |
+| [Record](../verbs/record.md)                      | Yes           |
+| [Redirect](../verbs/redirect.md)                  | Yes           |
+| [ResumeRecording](../verbs/resumeRecording.md)    | Yes           |
+| [SendDtmf](../verbs/sendDtmf.md)                  | Yes           |
+| [SpeakSentence](../verbs/speakSentence.md)        | Yes           |
+| [StartRecording](../verbs/startRecording.md)      | Yes           |
+| [StopRecording](../verbs/stopRecording.md)        | Yes           |
+| [Transfer](../verbs/transfer.md)                  | No            |
+
+It is important to note that no other BXML verbs may be specified after a Transfer Answer Event is called.
```

Comment:
I know you didn't write this, but what does this mean? Does it still apply?

Reply:
I believe so? I think it's just warning you if someone answers the transfer, no BXML after the transfer is going to be executed since it goes to whatever endpoint is specified as the transferAnswerURL.

Reply:
I would just remove it. That's what the table is saying